apps/web-portal/config/index.mjs

```diff
@@ -0,0 +1,10 @@
+import confme from 'confme'
+import path from 'path'
+
+// eslint-disable-next-line no-underscore-dangle
+const __dirname = path.resolve('')
+
+export const config = confme(
```

Comment:
Is it integrated with the application?  because I can't find a file which gets config and passed to `WebPortalApp`

Reply:
it is in apps/web-portal/index.mjs